lms/djangoapps/discussion_api/api.py

```diff
@@ -266,6 +275,7 @@ def get_thread_list(request, course_key, page, page_size, topic_id_list=None, te
     course = _get_course_or_404(course_key, request.user)
     context = get_context(course, request)
     query_params = {
+        "user_id": unicode(request.user.id),
```

Comment:
A user is required to get the state of unread/unanswered questions since those fields are unique per user.